### ISSUE TITLE
Unable to manage account in Cydia

### ISSUE THREAD

**Issue by compugeek2** [pwn20wndstuff/Undecimus#1142]
you are unable to click on any links on Cydia’s main page.  Reinstalling Cydia doesn’t resolve the issue.  Installing tweaks and everything else works.

**Comment by Tizioincognit0** [comment]
In all tweaks' pages, I have the same problem also with other options like “Developer Twitter”, “Change package settings” and “Filesystem content”. Because of this, I can’t ignore updates of any tweak.
This bug can’t be caused by a tweak because it is present also in safe mode and no-tweaks mode.

**Comment by Henkann** [comment]
Same here, cant tap on any of the links on cydia homepage, waiting for a fix :)

**Comment by WCHighfill** [comment]
Same here. I would like to be able to see the tweaks/apps I bought to see if I can reinstall them

**Comment by Tizioincognit0** [comment]
I’ve found a fix: a slow drag from the right side and letting go at which point you should be able to see the info pertaining to that tab.

**Comment by Henkann** [comment]
Nice fix, actually works if you have the patience!

**Comment by Baaleos** [comment]
Note - the fix mentioned here does not fix the issue of 'not being able to manage Cydia account'
If I'm not mistaken - the Cydia store closed down - so am I right in saying that we are unable to login and get our purchase history?

At the moment, I am prevented from logging in with facebook - does the same happen for others?

**Comment by Tizioincognit0** [comment]
No, my fix fixes the bug described by compugeek2 in the first comment which is related to the not working buttons.

**Comment by magicman0978** [comment]
You can tap and swipe to enter the links on the Cydia home page which is a work around not a fix. I’m having the same problem. The other problem some people are having is that once’s you get into the manage account page and try to log in it just gives you a page that says wait and stays there indefinitely.

**Comment by koiatoe** [comment]
Its super frustrating:( however could it be related to old cydia data put back on device after restore?

**Comment by Tizioincognit0** [comment]
> Its super frustrating:( however could it be related to old cydia data put back on device after restore?

I've this issue even after a full restore via iTunes.

**Comment by Tizioincognit0** [comment]
I’ve this bug also after the last Cydia Installer update (1.1.32~b19).

**Comment by badger200** [comment]
**I just found a solid gold workaround!!**

Press the icon on the left side of the "Filesystem Content" or "Change Package Settings" bar, and IMMEDIATELY drag the icon to the right about 2cm. Works EVERY TIME!! I was able to get into Change Package Settings literally 20 times in a row, 100% success rate.

Just don't hold down so long that it becomes a long press.

iOS 12.4 iPad Pro with stock Cydia from unc0ver 3.5.6.
